### DISCUSSION TITLE
Right Arrow - Skip to Next Message

### DISCUSSION THREAD

**Discussion by toddzim33** [elk-zone/elk#2878] [Ideas]
Inoreader has a nice keyboard shortcut where the right arrow skips to the next message. Makes scrolling though the timeline very slick. Any chance to add??

**Comment by shuuji3** [comment]
We have a pending PR (https://github.com/elk-zone/elk/pull/2739) for similar functionality for `j`/`k` keyboard shortcut. So I think we could propose adding arrow keys as an additional shortcut once implemented.

**Comment by toddzim33** [comment]
Fantastic, thank you!

TZ
